app/gui/src/main/java/org/mobilitydata/gtfsvalidator/app/gui/Main.java

```diff
@@ -0,0 +1,57 @@
+package org.mobilitydata.gtfsvalidator.app.gui;
+
+import com.google.common.flogger.FluentLogger;
+import java.awt.Desktop;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The main entry point for the GUI application.
+ *
+ * Compared to the CLI jar, this entry point is designed to be packaged as a
+ * native application to be run directly by the user.
+ *
+ * TODO(bdferris): Follow up work will add a minimal UI for selecting the input
+ * GTFS and potentially the output directory.
+ */
+public class Main {
+  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+
+  public static void main(String[] args) {
+    // On Windows, if you drag a file onto the application shortcut, it will
+    // execute the app with the file as the first command-line argument.  This
+    // doesn't appear to work on Mac OS.
+    if (args.length != 1) {
+      System.exit(-1);
+    }
+
+    Path workingDirectory = null;
+    try {
+      workingDirectory = Files.createTempDirectory("GtfsValidator");
```

Comment:
I also think that the directory of the input GTFS zip file may be a better option. 

Although, maybe I'm missing a point here. Why wouldn't the be organized in a single directory if the GTFS files aren't? Do you mean if there is multiple version of the GTFS zip file in the same directory rather than in subdirectories?

Reply:
I think it's worth noting that I'd still like to support validation against a URL, so we might still need to find a directory for report output independent of the feed location.

I can think of a couple of additional options:
1) We create an explicitly named directory off the user's home directory to store reports.  You'll notice that I also added support for diagnostic logging to a $HOME/gtfs-validation.log file, so could be worth consolidating all artifacts in the same place?
2) We ask the user to pick an output directory the first time (or every time) that validation runs.  Or we at least remember their previous choice and pop-up a dialog with defaults and ask them to click "Validate" before things kick off.

Reply:
>Why wouldn't the be organized in a single directory if the GTFS files aren't?

If outputting to temp directly everything gets dumped to that same directory. But if we use the GTFS directory, then it changes depending on where your GTFS zip file is.

>I think it's worth noting that I'd still like to support validation against a URL, so we might still need to find a directory for report output independent of the feed location.

I actually like a combination of 1 and 2 above.

We ask them to pick output, but default to an explicitly named directory off user's home directory. Then save whatever they use for future executions. This aligns with the normal Windows application install experience so it should be familiar to most users, and even non-technical ones.

Reply:
Cool, that sounds good for me.

For now, I've updated the code to use a default output directory ~/GtfsValidator per a2f0567e1f8686e7bf9410a939897b0ddad69272.  This will be used by default for both output reports and logs.

In a follow-up PR where we add in the GUI, we'll allow the user to confirm or change this default and remember their choice for subsequent runs.